changelog/fragments/run-bundle-upgrade-bugfix.yaml

```diff
@@ -0,0 +1,25 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For run bundle-upgrade, generate just the extra FBC of the bundle instead of rendering the entire index and appending bundle contents to it.
```

Comment:
How can we test this one to check that the PR fix?
What is the image/data used to do the test?

Reply:
These are the images I used for testing: 
run bundle with bundle `quay.io/olmqe/upgradeindex-bundle:v0.1` and index image `quay.io/olmqe/nginxolm-operator-index:v1`. 

And `run bundle-upgrade` with `quay.io/olmqe/upgradeindex-bundle:v0.2`